kratos/python_scripts/KratosUnittest.py

```diff
@@ -106,10 +106,8 @@ def GetValErrMsg(idx_1, idx_2):
 def skipIfApplicationsNotAvailable(*application_names):
     '''Skips the test if required applications are not available'''
     required_but_not_available_apps = GetNotAvailableApplications(application_names)
```

Comment:
```suggestion
    required_but_not_available_apps = GetNotAvailableApplications(*application_names)
```

I think this can solve it, can you try